book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+
+  att_reader :rentals
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] As stated in the project requirement  A book has many rentals: you should be implemented **A method for adding rentals**, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association Just like this :- 
```
def add_rental(person, date)
   Rental.new(date, self, person)
  end
```